### ISSUE TITLE
Not longer mantained?

### ISSUE THREAD

**Issue by davidpelayo** [nnajm/orb#87]
I wonder whether this repo is being maintained or not.

In case not, I will fork and publish it on my own so we can go maintaining this cool idea.

Any feedback?

**Comment by davidpelayo** [comment]
I started the new fork on this repo: https://github.com/davidpelayo/orb-latest.

**Comment by davidpelayo** [comment]
@nnajm are you still up to maintain this repo? Can you transfer the ownership of the domain name to me so I can continue with this project? Thanks in advance.

**Comment by ariutta** [comment]
@davidpelayo, any chance you'd want to collaborate with @super-cache-money? There's already a [forked repo](https://github.com/super-cache-money/orb) and corresponding [NPM package](https://www.npmjs.com/package/orb-fork). It'd be great to see development coalesce around one fork as an [OPEN Open Source Project](http://openopensource.org/).

**Comment by davidpelayo** [comment]
@ariutta I would agree with that. However, after being thinking a while, I believe it's better to rewrite the solution from the scratch [to start a new one](https://news.ycombinator.com/item?id=14512995). [The issue I raised](https://github.com/davidpelayo/orb-latest/issues/4). What do you guys think?

**Comment by thecodejack** [comment]
Agreed with @davidpelayo. I have improved current version of orb to match excel like features for my work. But I am still facing lot of issues. For example, orb stucks when it is having 3 cols,3 rows and 4 aggregate functions with 10,000 rows of complex data. And there is no easy fix on orb. We need proper open source project. I can help you guys as well.

**Comment by ariutta** [comment]
I'm fine with a rewrite. Could we keep the same API?

**Comment by super-cache-money** [comment]
@davidpelayo rewrite sounds great! It's such a shame that this has stagnated. Please ignore my fork, I haven't/can't work on it at all. I can transfer the npm package rights to you if you'd like.

**Comment by davidpelayo** [comment]
@super-cache-money @ariutta I even thought of [renaming the package (already created a repo for it)](https://github.com/binomads/orbitable). We can start with this current package as a legacy solution, so we can go fixing and iterating it for the community, if needed.
In parallel, starting a new fresh solution with the new API, while consistently founding the contributors circle and establish the guidelines.

Who is willing to **really contribute** to the development?
Please contact me so we can bootstrap this thing asap.

**Comment by ariutta** [comment]
Realistically, I've probably only got time for testing and bug-fixing at the moment.

**Comment by ariutta** [comment]
If you're doing a rewrite, you may be interested in taking a look at the library [tad](https://github.com/antonycourtney/tad). It's currently packaged as an Electron app, but much of the code would be useful for an in-browser pivot table.

**Comment by thecodejack** [comment]
@davidpelayo I can help over the weekends.

**Comment by turnerniles** [comment]
Hi @nnajm, @davidpelayo, @jsegura, @ariutta, @super-cache-money, @cohenoshri and all. We are facing the challenge of pivoting "large" amounts of data in browser. And want a similar experience to [orb](https://github.com/nnajm/orb) and [PivotTable.js](https://github.com/nicolaskruchten/pivottable). We are working on [**react-virtualized-pivot**](https://github.com/turnerniles/react-virtualized-pivot) and the pivot engine, [**quick-pivot**](https://github.com/pat310/quick-pivot). We're still trying to reach feature parity with orb and have come quite a long way. Let me know what you think. We would appreciate any help and feedback if you're interested in the project. Please open issues as you see fit. We have a ways to go.

**Comment by cohenoshri** [comment]
I am interested in contributing, I started converting that project to reactJS but had to let it go due to time constraints.

**Comment by turnerniles** [comment]
@cohenoshri @all, we've made a lot of progress on [**react-virtualized-pivot**](https://github.com/turnerniles/react-virtualized-pivot). Let us know what you think!
![image](https://user-images.githubusercontent.com/8146241/29742823-06c2adf8-8a54-11e7-96ed-18466e853482.gif)

**Comment by super-cache-money** [comment]
Wow, looks great!
Arjun Khoosal | https://www.kandua.com

**Comment by DavidIMcIntosh** [comment]
Hi @turnerniles.  How would you describe the state of usability of this  react-virtualized-pivot?  I am willing to do some work over the long haul, but I need something that is at least currently functional - I have a deliverable due in a month, and need at least basic functionality.

**Comment by turnerniles** [comment]
@DavidIMcIntosh It's fully functional at this point and we are working on additional features. Please check out the demo [here](https://turnerniles.github.io/react-virtualized-pivot) and leave feedback / open issues on the [github](https://github.com/turnerniles/react-virtualized-pivot)!

**Comment by DavidIMcIntosh** [comment]
@turnerniles Thanks for the quick response.  Will do.  Is there a list of features that need working on in the repo?

**Comment by turnerniles** [comment]
@DavidIMcIntosh We're sort of just expanding the feature set as issues/requests come in. A well defined list of needed features and/or a roadmap is something we should add. We've just added row totals and we're working to add column totals, predefined pivots and allowing the pivot to return its state. Please open issues as you see fit for any additional features. I think in general we may have to rethink the drawer/menu a bit...

**Comment by davidpelayo** [comment]
Stale. Closing
